schemas/org/openstax/ec/created_highlight/v1/avro_builder.rb

```diff
@@ -0,0 +1,12 @@
+namespace 'org.openstax.ec'
+
+import 'types/uuid'
+import 'types/date_time'
+
+record :created_highlight_v1 do
+  required :user_uuid, :uuid
+  required :contents, :string
+  required :location, :string
+  required :color, :string
```

Comment:
Add annotation?  Should we make these fields map more to what we have directly in highlights (source type, etc)?

Reply:
we should discuss this.  this would be a general pattern to follow.  in this case, highlights, here we have a created. we'll prob. want a updated, a deleted, etc.  and then for other entities (like annotations, adding books, adding courses, etc etc).  doesnt it seem like a not-so-great idea to be storing large amounts of text w/ stuff like this?  (annotations, and contents).  can we simply store a HL id that could later, relationally, be linked back to the contents?

Reply:
https://openstax.slack.com/archives/C01498TCQ77/p1603470504331500?thread_ts=1603405353.328100&cid=C01498TCQ77